main.cpp

```diff
@@ -13,75 +13,571 @@ What you create in this project will be used as the basis of Project 5 in the co
+
+    struct SoccerTeam
+    {
+        struct Manager
+        {
+            ...
+        };
+
+        // train their skills
+        void trainPlayerSkills(Manager managerA);
+    
+        Manager teamManager;
+    };
+
+    It does not make sense to pass in a new Manager whenever you are going to train your team players.
+    Soccer teams have ONE manager.
+
+    a much more relevant usage would be adding a member function that hires a new manager:
+
+    struct SoccerTeam
+    {
+        struct Manager
+        {
+            ...
+        };
+
+        void hireNewManager(Manager newManager);
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
-*/
+        Manager teamManager;
+    };
+
+    We can safely assume that the 'Manager' instance that is being passed in will be replacing the current 'teamManager' variable without looking at any other code.
+    This is because the function name and function argument clearly indicate what they are/what they do.
+
+    Your function names and parameter names should make LOGICAL SENSE.
+    Readers of your code should be able to INTUITIVELY understand what your function implementations will do without actually seeing the implementations.
+
+    Keep this in mind when you define your UDTs in this project part.
+    */
+};
+
+
+/*
+Thing 1) Coffee shop
+5 properties:
+    1) the number of baristas (int)
+    2) the number of coffee machines (int)
+    3) the number of registers (int)
+    4) the number of tables (int)
+    5) the types of coffee (std::string)
+3 things it can do:
+    1) make a coffee
+    2) grind coffee beans
+    3) modify customer names
+ */
+
+struct CoffeeShop
+{
+    // 1) the number of baristas (int)
+    int numBaristas = 2;
+    // 2) the number of coffee machines (int)
+    int numCoffeeMachines = 4;
+    // 3) the number of registers (int)
+    int numRegisters = 2;
+    // 4) the number of tables (int)
+    int numTables = 12;
+    // 5) the types of coffee (std::string)
+    std::string brewMethod = "pour over";
+
+    struct Coffee
+    {
+        std::string type = "columbian";
+        std::string roast = "dark";
+    };
+
+    struct Customer
+    {
+        std::string name = "Customer 1";
+        float customerPhoneNumber = 15555555555.0f; //replit suggested 'long' be used, but that's not in the lesson
+        float customerID = 1.0f; // I'll also float this then
```

Comment:
Use an integer, not a float to store this type of information
- [ ] revise

Reply:
changing to "int customerPhoneNumber = 15555555555;" brings up the warning about implicit conversion from long to int